### ISSUE TITLE
nicehash

### ISSUE THREAD

**Issue by whitindllc** [RainbowMiner/RainbowMiner#2899]
Looks like it is moving its servers to another county and doing a whole re-build

Dear NiceHash Miner,

We are excited to [announce](https://mailer.nicehash.com/l/ZAVkdNV80IGOOzu6mSdHbA/UdGdG1kW28N763mEyiITXKYw/GndBnIJSj6GFqhQbdKGYqg) the date of our move to Switzerland.

On the 25th of November 2024, we will start a transition period. In this period, users will be transitioning from NiceHash LTD (British Virgin Islands) to NiceHash AG (Switzerland). 

This transition period will last until the 9th of December 2024.
What to expect?

After logging into your NiceHash account following the transition, you will need to agree to our updated terms and conditions. Once completed, you can continue using the platform as usual.

Not agreeing with new terms and conditions, will result in users not being able to access NiceHash services after the transition period has ended.

After transitioning, all users will receive new [Organization IDs](https://mailer.nicehash.com/l/ZAVkdNV80IGOOzu6mSdHbA/nTkhug1VBpjYX5ZOxQ9KZQ/GndBnIJSj6GFqhQbdKGYqg) for all organizations. Old organizations will remain visible in view-only mode for generating reports.

With the transition to new organizations, most of your account information will move over automatically. However, some details will need to be reconfigured manually.
What will be transferred automatically?

    Available Funds 
    Unpaid Balance
    Mining Addresses
    Mining Statistics
    Lightning Network Addresses 
    KYC Information
    Pools for Hashrate Marketplace
    Referral Codes and Affiliates
    NFC Cards

What will NOT be transferred?

These details require user action:

    Withdrawal Addresses - Will need to be re-entered.
    API Keys - Old API keys will no longer be valid; users must create new ones.
    Deposit Addresses* – Previous deposit addresses will be invalid, and users must request new ones.

*Funds deposited to old addresses can be transferred later. Users will have a button to initiate this transfer, along with a prompt to provide the necessary source-of-funds information.

Thank you for being part of NiceHash as we move forward into this exciting new era.

Please note that dates are subject to change due to unforeseen circumstances.

Kind regards,

NiceHash Team

**Comment by RainbowMiner** [comment]
Yes, in fact at some time from the 25th November you will have to create yourself new API credentials and change the Nicehash pools.config.txt settings for organization ID, API key and API secret and wallet address.

**Comment by rizkyaulya** [comment]
> Yes, in fact at some time from the 25th November you will have to create yourself new API credentials and change the Nicehash pools.config.txt settings for organization ID, API key and API secret.

I am just doing this and my RIGs mining to nicehash again. but the balance doesn;t showing up, wallet and ballance API already open. what to do?

**Comment by RainbowMiner** [comment]
Could you please send me the most current log file RainbowMiner_2025-01-23.txt in folder "Logs" by email to info@rbminer.net ?

**Comment by rizkyaulya** [comment]
> Could you please send me the most current log file RainbowMiner_2025-01-23.txt in folder "Logs" by email to [info@rbminer.net](mailto:info@rbminer.net) ?

File sent.

**Comment by RainbowMiner** [comment]
Ok, the balance API responds with code 401 (=Unauthorized). So you will have to double check all three critical values OrganizationID, API_Key and API_Secret in pools.config.txt. 
- OrganizationID and API_Key need to be exactly 36 characters long (make sure to include all the "-"), the format is xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx where "x" is a character [0-9a-f]
- API_Secret need to be exactly 72 characters long (again, make sure to include all the "-"), the format is xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxxxxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx where "x" is a character [0-9a-f]

If all values are all right, check the permissions for the API-Key set:
- "Wallet-Permissions -> View balances, wallet activities and deposit addresses" needs to enabled.
